src/Client.php

```diff
@@ -671,7 +677,8 @@ protected function getParcelData(
         ?string $customsInvoiceNumber,
         ?int $customsShipmentType,
         ?array $items,
-        ?string $postNumber
+        ?string $postNumber,
+        bool $applyShippingRules = false
```

Comment:
I think `false` is the right default here because it will not change API behavior for extending classes (which I doubt there are any though).